src/modules/preview-site/components/preview-site-row.tsx

```diff
@@ -92,7 +92,7 @@ export function PreviewSiteRow( {
 	return (
 		<div className="self-stretch flex-col">
 			<div className="flex items-center px-8 py-6">
-				<div className="w-[51%]">
+				<div className="w-[51%] overflow-hidden ltr:pr-4 rtl:pl-4">
```

Comment:
Let's use [padding inline end](https://developer.mozilla.org/en-US/docs/Web/CSS/padding-inline-end)

```suggestion
				<div className="w-[51%] overflow-hidden pe-4">
```

Reply:
Oh, nice! Thanks, Antonio. I will commit the change.